### PR TITLE
Where am i working?

### PR DESCRIPTION
**Pull request by sahibak**
I wanted to provide the software location of github.

@yagnik

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 Repository 1 Branch readme
 
 Hi, I am trying to learn coding.
+
+Trying desktop.
```
Comment (yagnik) [inline README.md]: lets do capital desktop

Comment (sahibak) [reply README.md]: I don't think so.

Comment (yagnik) [reply README.md]: for the sake of learning lets do it

Comment (yagnik) [inline README.md]: for learning lets change it to capital desktop

**Pull request merged**